.gitignore

```diff
@@ -109,7 +109,7 @@ app/drucker_pb2_grpc.py
 
 # sqlite
 app/db.sqlite3
-app/db.test.sqlite3
+app/test/db.test.sqlite3
```

Comment:
I changed it because 
when I run unittest ([commands in README](https://github.com/drucker/drucker-dashboard/pull/45/files#diff-069aca75954344cbe412c9b31002d366R21)), db file is generated in `app/test` and it's not a big deal.

Reply:
`app/test/db.sqlite3` is generated after running `python -m unittest`.

Reply:
@keigohtr 
I fixed how to run test for now.
I set to ignore some warnings (because it will be fixed when updating dashboard to upload to PyPI. I will remove the ignore statement when the update is done)